### DISCUSSION TITLE
Don't show admin panel outside your network

### DISCUSSION THREAD

**Discussion by RandomBloke85** [dani-garcia/vaultwarden#3533] [Ideas]
Hello

I've just finished installing Vaultwarden. Thank you for your work!

I wonder if it's possible to use the admin site only internally? If not, maybe that would be a feature for a future release? I would prefer not to expose the admin site to the open web.

Thank you

**Comment by bokkabonga** [comment]
We did this as part of our Reverse-Proxy config. In my case apache:

  <Location /admin>
      Order Deny,Allow
      Deny from all
      Allow from *your Network*
  </Location>

**Reply by RandomBloke85** [reply]
I solved it exactly llike this. Thank you

**Comment by CamelT0E** [comment]
i have also requested it before in #2957. I would think it would be nice if you could separate the two better.
The easiest way would be with a different admin GUI port instead of /admin.

I, like many, use a Synology Diskstation. The reverse proxy does not have advanced settings to block /admin. En further reverse proxy causes problems with the occupied ports.

Maybe the Devs will think about it again. I and others would be very happy about it and increase security. The deactivation of admin-webfi would be omitted.

**Comment by mkopnsrc** [comment]
Have a look at this, I've posted screenshot from NPM,

https://github.com/dani-garcia/vaultwarden/discussions/2547#discussioncomment-6041290

**Reply by CamelT0E** [reply]
Synology Reverse Proxy has no location

**Comment by CamelT0E** [comment]
I tried it again with another reverse proxy on the diskstation. This only works partially in conjunction with macvlan. But this creates other problems again.

I'm still hoping for another solution on how to better disconnect the admin panel without removing the environment every time. NPM is not a proper option on the Synology Diskstaion. There are many users who use this NAS.

What is wrong with a different admin port?

**Reply by BlackDex** [reply]
The argument that there are many users who use that NAS is invalid since other users are able to solve the issue using NPM (Nginx Proxy Manager)

The main problem is, is that we would need to add a lot of logic and split stuff to just make it able to work on a different port.
Which, in the end only causes confusion for other users. I'm not in favor of adding something like that.
Your use-case is what reverse proxy are intended for.

**Reply by CamelT0E** [reply]
Unfortunately, the reverse proxy is very slim on the diskstation. Another one is a problem because of the busy ports. macvlan makes more problems. Maybe there is another solution.

**Reply by BlackDex** [reply]
As already mentioned before by someone else **Use the Nginx Proxy Manager (NPM)**.
Here is a link to an online tutorial (which i found by searching) https://mariushosting.com/how-to-install-nginx-proxy-manager-on-your-synology-nas/

After that i would suggest to read: https://github.com/dani-garcia/vaultwarden/discussions/3533#discussioncomment-6041308

When you can install docker containers, there are almost no limits 😉

**Reply by CamelT0E** [reply]
installing it is not the problem. note the changed ports!
A reverse proxy should listen on ports 80/443.
These ports are reserved at Synology and are blocked for all network cards and networks. this is not well solved at synology.

Even in the macvlan you can not use the ports. In addition, there is also the separation from the macvlan to the host and that the containers in the macvlan do not have internet, the workaround does not work for me. the block in NPM works and is also faster, but unfortunately not to be used properly on the Synology DiskStation otherwise I had already used it for a long time.
